Utils/motion.hpp

```diff
@@ -28,7 +35,7 @@ double degree2arc(double deg) {
 }
 
 double arc2deg(double arc) {
-  return 180 * M_PI / arc;
+  return 180 * arc / M_PI;
```

Comment:
why is it 180 degree, not 360?

Reply:
M_PI corresponds to 180 degrees, not 360.